apps/pushy/src/pushy_node_stats_scanner.erl

```diff
@@ -48,7 +48,7 @@ start_link() ->
     gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
 
 init([]) ->
-    _Seed = random:seed(erlang:now()),
+    _Seed = random:seed(erlang:monotonic_time(milli_seconds)),
```

Comment:
From the docs it looks like if we want the same format we should use `erlang:system_time/0`

http://erlang.org/doc/apps/erts/time_correction.html#Dos_and_Donts